spec/translation_quality/tqc_common.yaml

```diff
@@ -164,6 +171,116 @@ components:
           type: string
           description: Unique identifier of corresponding group.
 
+    regexRuleGroupModel:
+      type: object
+      properties:
+        groupUid:
+          type: string
+          description: Smartling rule group identifier of the translation
+          example: "ce6d6b5dbc12"
+        ruleUid:
+          type: string
+          description: Smartling rule identifier of the translation
+          example: "ce6d6b5dbcs2"
+        targetLocaleIds:
+          description: The list of target locales.
+          example:
+            - "ru-RU"
+          items:
+            type: string
+          type: array
+        severityLevelCode:
+          type: string
+          $ref: '#/components/schemas/enum/severityLevelCodeEnum'
+          description: severity level code
+          example: "MEDIUM"
+
+    regexRuleModel:
+      type: object
+      properties:
+        ruleName:
+          type: string
+          description: Rule name
+          example: "ruleName"
+        severityLevelCode:
+          type: string
+          $ref: '#/components/schemas/enum/severityLevelCodeEnumCopyForRequest'
+          description: '[Severity level code](#operation/getTranslationQualityCheckSeverityLevels)'
+          example: "MEDIUM"
+        ruleTypeCode:
+          type: string
+          $ref: '#/components/schemas/enum/ruleTypeCodeEnumCopyForRequest'
+          description: '[Rule type code](#operation/getRegexQualityCheckRuleTypes)'
+          example: "FORBIDDEN"
+        flavorCode:
+          type: string
+          $ref: '#/components/schemas/enum/flavorCodeEnum'
+          description: Flavor code
+          example: "JAVA"
+        originalPattern:
+          type: string
+          description: Regular expression to evaluate against the source string.
+          example: "@mail"
+        translationPattern:
+          type: string
+          description: Regular expression to evaluate against the translation.
+          example: "@gmail"
+        caseInsensitive:
+          type: boolean
+          description: Enable case insensitive search
+          example: true
+        description:
+          type: string
+          description: Human readable description of the role; shown in the the dashboard UI.
+          example: "rule description"
+        warningMessage:
+          type: string
+          description: Human readable warning message appended to the standard message that is shown in the CAT tool when the check fails.
+          example: " test warning message"
+      required:
+        - ruleName
+        - severityLevelCode
+        - ruleTypeCode
+        - flavorCode
+        - originalPattern
+        - translationPattern
+        - caseInsensitive
+
+    regexRuleWithRuleUid:
+      allOf:
+        - type: object
+          properties:
+            ruleUid:
+              type: string
```

Comment:
inject from common file

Reply:
it's already in tqc common file I don't prefer to move it into global common file